rust/arrow/src/array/builder.rs

```diff
@@ -301,6 +324,21 @@ impl BufferBuilderTrait<BooleanType> for BufferBuilder<BooleanType> {
         Ok(())
     }
 
+    fn append_n(&mut self, n: usize, v: bool) -> Result<()> {
+        self.reserve(n)?;
+        if v {
+            unsafe {
+                bit_util::set_bits_raw(
+                    self.buffer.raw_data() as *mut u8,
```

Comment:
This can be `raw_data_mut` instead, right?

Reply:
Changed, and fixed the others in the same file